MkDocs/apm_8.8/docs/matrix.md

```diff
@@ -0,0 +1,100 @@
+# Asset Matrix
+
+Maximo HP Utilities includes matrix view for determing the distribution of assets based on scores and then take action to avoid imminent failures.
+
+In this exercise the Reliability Engineer uses HP Utilities to:
+	
+1. [View Matrix for Criticality and End of life, Criticality and Risk, Criticality and Health](#view-matrix-for-criticality-and-end-of-life-criticality-and-risk-criticality-and-health)
+2. [View the drilled in assets list and take action](#view-the-drilled-in-assets-list-for-one-distribution)
+3. [Config Matrix version](#config-matrix-version)
+
+
+## Pre-requisites 
+
+This lab requires the following
+
+- A working Maximo Application Suite (MAS) environment with Maximo Manage, or another Enterprise Asset Management (EAM) system, Maximo Health, Maximo Health and Predict – Utilities (HPU), and Maximo Optimizer installed
```

Comment:
A working Maximo v8.8 environment?  We should be specific about which version it works with.

Reply:
added the version number